phpstan.dist.neon

```diff
@@ -27,3 +27,8 @@ parameters:
             count: 18
     tipsOfTheDay: false
     tmpDir: dev-tools/phpstan/cache
+
+    strictRules:
+        # False positives related to `AbstractFixer::$configuration` and accessing bool config options
+        # see: https://github.com/phpstan/phpstan/discussions/9686
+        booleansInConditions: false
```

Comment:
Won't that skip any future issues like `!$string` or `if (count($array))`?

Reply:
Yes, it will, but at this point almost all issues are related to configuration, so I thought we can get rid of it and decrease amount of issues in baseline. I wanted to provide actual fix later. But maybe we just should keep those entries in baseline and keep this flag enabled, to prevent new issues?

Anyway, I started to add bool conditions like `true === ...`, also tried defining `$configuration` with proper array shape in the fixer, but both are only workaround for the PHPStan and I believe it's not a good direction. Best thing would be to infer proper array shape from fixer's config definition, like I asked [here](https://github.com/phpstan/phpstan/discussions/9686).